{
  "path": "/documentation/foundation/urlerror/3329607-networkunavailablereason",
  "type": "Instance Property",
  "name": "networkUnavailableReason",
  "desc": "The reason the network was unavailable for a task.",
  "items": [],
  "declaration": "var networkUnavailableReason: URLError.NetworkUnavailableReason? { get }"
}